docs/Developers/Guides/whitelistAddress.md

```diff
@@ -0,0 +1,88 @@
+# Getting Read Access to Contracts
+
+Our contracts are read-protected by a whitelist, meaning you won't be able to read them on-chain without your address being added to the whitelist.
+
+:::info
+
+On Testnet, the [SelfKisser contract](https://github.com/chronicleprotocol/self-kisser) allows users to whitelist themselves, whimsically termed 'kiss' themselves.
+
+On Mainnet, to get access to production  Oracles, please open a ticket on [Discord](https://discord.gg/gJzFthdD) via the 🆘｜support channel. 
+
+:::
+
+# Whitelisting Yourself on Testnet and Reading from Smart Contracts
+## Option 1: Using the CLI with `cast`
+
+### Whitelisting Using the CLI
+
+[Cast](https://book.getfoundry.sh/reference/cast/cast) is a command-line interface (CLI) tool developed by the Foundry team to streamline interactions with Ethereum and other EVM-based chains. Whether you're making smart contract calls, sending transactions, or retrieving chain data, Cast can help you with that.
+
+```bash
+$ cast send 0x0dcc19657007713483a5ca76e6a7bbe5f56ea37d "selfKiss(address)()" <oracle address you want you use> --rpc-url $rpc
+```
+
+Where `0x0dcc19657007713483a5ca76e6a7bbe5f56ea37d` is the SelfKisser contract address.
+
+If you are looking for an Oracle address to quickly test out, feel free to use: Chronicle_ETH_USD_3	`0xdd6D76262Fd7BdDe428dcfCd94386EbAe0151603`.
+
+You can then verify your whitelist status by running:
+```bash
+$ cast call <oracle address> "tolled(address)" <your address> --rpc-url $rpc
+```
+`0x0000000000000000000000000000000000000000000000000000000000000001`
+
+A response of 1 (true) confirms your whitelisted status.
+
+### Reading from Smart Contracts using the CLI 
+Once you are whitelisted, interacting with the oracle is easy. The `read()` function returns a uint value representing the data you are trying to get:
+
+```bash
+$ cast call 0xc8A1F9461115EF3C1E84Da6515A88Ea49CA97660 "read()(uint)" --rpc-url $rpc
+```
+
+Response:
+`1656870000000000045440`
```

Comment:
we have other "read" functions too, but maybe not necessary for the scope of this doc? i.e. `tryRead`, `tryReadWithAge` etc

Reply:
Thank you for bringing this up. I will leave it out of the scope for this one, but definitely integrate into a future one.